airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -46,7 +46,7 @@ def request_params(
 
         # Stripe default pagination is 10, max is 100
         params = {"limit": 100}
-        for key in ("created[gte]", "created[lte]"):
+        for key in ("created[gt]", "created[lt]"):
```

Comment:
No, this is the original connector's format

Reply:
@ssbeefeater Not sure what we used in blendo code, but based on my experience of stripe data, some time created timestamp for multiple records can be same. I am guessing for that reason they might have added `gte/lte`. As in with pagination limit of 100, its possible we have got 100 records with `created < T`. And still there are some records left to be fetched in next batch with created value T. And say this task finishes, and next task run start fetching for `created > T`. Then it will miss some records.

I would suggest to read more about this field and possible records with same value for created in stripe doc before finalizing solution. I am thinking `gte/lt` should cover all scenarios but have not thought thoroughly.